README.md

```diff
@@ -109,4 +116,5 @@ How is `git pull` related to `git fetch`?
 
 ```text
 Your answer here.
+`git pull` will `git fetch` any changes from the remote repository and merge them with the local branch.
 ```
```

Comment:
Yes! git fetch is just checking for changes or commits from the remote repo that you don't currently have locally. While a git pull is combing a git fetch and git merge into one!